packages/types/src/discord/user.ts

```diff
@@ -30,12 +20,22 @@ export interface DiscordUser {
   system?: boolean
   /** Whether the user has two factor enabled on their account */
   mfa_enabled?: boolean
+  /** the user's banner, or null if unset */
+  banner?: string | null
+  /** the user's banner color encoded as an integer representation of hexadecimal color code */
+  accent_color?: number
```

Comment:
https://discord.com/developers/docs/resources/user#user-object

```suggestion
  accent_color?: number | null
```